src/microbe_stage/MicrobeStage.cs

```diff
@@ -384,9 +384,12 @@ public override void _Process(float delta)
             }
         }
 
-        // Start auto-evo if not already and settings have auto-evo be started during gameplay
-        if (Settings.Instance.RunAutoEvoDuringGamePlay)
+        // If auto save not wanted, start auto-evo if not already start,
```

Comment:
I think the proper word form here is `started`
Also I think it should be a bit more abstract "If stage entry finished, --"
instead of explicitly just saying that auto save not wanted, which is a bit unclearer on when this will be triggered.

Reply:
Okay, I'm sorry I'm not very good in english but I try to do my best to make a comprehensive comment.